src/redis-cli.c

```diff
@@ -6935,17 +6935,13 @@ static int clusterManagerCommandCreate(int argc, char **argv) {
     int replicas = config.cluster_manager_command.replicas;
     int masters_count = CLUSTER_MANAGER_MASTERS_COUNT(node_len, replicas);
     if (masters_count < 3) {
-        clusterManagerLogErr(
-            "*** ERROR: Invalid configuration for cluster creation.\n"
-            "*** Redis Cluster requires at least 3 master nodes.\n"
-            "*** This is not possible with %d nodes and %d replicas per node.",
-            node_len, replicas);
-        clusterManagerLogErr("\n*** At least %d nodes are required.\n",
-                             3 * (replicas + 1));
-        return 0;
+        int ignore_force = 0;
+        if (!confirmWithYes("Redis Cluster requires at least 3 master nodes for "
+                            "automatic failover, are you sure?", ignore_force))
```

Comment:
More correct grammar?

```suggestion
        if (!confirmWithYes("Redis Cluster requires at least 3 master nodes for "
                            "automatic failover. Are you sure?", ignore_force))
```